crates/superposition_types/src/custom_query.rs

```diff
@@ -197,3 +198,46 @@ impl<'de> Deserialize<'de> for PaginationParams {
         })
     }
 }
+
+#[derive(Debug, Deserialize, Clone, Deref)]
+#[deref(forward)]
+pub struct StringArgs(
+    #[serde(deserialize_with = "deserialize_stringified_list")] pub Vec<String>,
+);
+
+pub fn deserialize_stringified_list<'de, D, I>(
```

Comment:
can we instead implement the `Deserialize` trait on `StringArgs` and write the logic in it instead of writing this as a function

Reply:
I tried Deserialize, it was more verbose, and all examples I found did it this way. Can we continue with this method for now?

Reply:
let me know if this works:
https://github.com/juspay/superposition/pull/280/files#r1865864524